chapters/intro/intro.md

```diff
@@ -0,0 +1,108 @@
+
+[development]: http://www.viewpoints-and-perspectives.info/home/viewpoints/
+
+A third recurring theme is _variability management_. Many of today's software systems are highly configurable. In such systems, different features can be enabled or disabled, at compile time or at run time. Using techniques from the field of [product line engineering][fospl], several of the chapters provide feature-based variability models of the systems under study.
+
+A fourth theme is [metrics-based evaluation of software architectures][bouwers]. Using such metrics architects can discuss  (desired) quality attributes (performance, scaleability, maintainability, …) of a system quantitatively. Therefore various chapters discuss metrics and in some cases actual measurements tailored towards the systems under analysis.
+
+
+## First-Hand Experience
+
+Last but not least, the chapters are also based on the student's experience in actually contributing to the systems described. As part of the course over 75 pull requests to the projects under study were made, including refactorings
+([Jekyll 3545](https://github.com/jekyll/jekyll/pull/3545),
+  [Docker 11350](https://github.com/docker/docker/pull/11350),
+  [Syncany 391](https://github.com/syncany/syncany/pull/391)), bug fixes
+([Docker 11018](https://github.com/docker/docker/pull/11018),
+  [Diaspora 5714](https://github.com/diaspora/diaspora/pull/5714),
+ [OpenRA 7486](https://github.com/OpenRA/OpenRA/pull/7486), [OpenRA 7544](https://github.com/OpenRA/OpenRA/pull/7544),
+ [Kodi 6570](https://github.com/xbmc/xbmc/pull/6570)), and helpful documentation such as a [Play Framework screen cast](https://www.youtube.com/watch?v=47TqFH8MXhk).
+
+Through these contributions the students often interacted with lead developers and architects of the systems under study, gaining first-hand experience with the architectural trade-offs made in these systems.
+
+## Feedback
+
+While we worked hard on the chapters to the best of our abilities, there will be plenty of omissions and inaccuracies.
+We value your feedback on any of the material in the book. For your feedback, you can:
+
+* Open an issue on our [GitHub repository for this book][dswa.io].
+* Offer an improvement to a chapter by posting a pull request on our [GitHub repository][dswa.io].
+* Contact [@delftswa][dswa.tw] on Twitter.
+* Send an email to Arie.vanDeursen at tudelft.nl.
+
+[dswa.io]: https://github.com/delftswa/delftswa.github.io
+[dswa.tw]: https://twitter.com/delftswa
+
+
+## Acknowledgments
+
+We would like to thank:
+
+* The additional members of the teaching team: Eric Bouwers, Nicolas Dintzner, Annibale Panichella, and Felienne Hermans;
+* Michael de Jong and Alex Nederlof who were instrumental in the earlier editions of this course.
+* All open source developers who helpfully responded to the student's questions and contributions;
```

Comment:
A little inconsistency in the line endings: not all of them end with a semicolon (personally I prefer periods, but semicolons are fine as well... but it should at least be consistent :blush:)